annotations/fenix/metrics/first_session.adgroup/README.md

```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - Performance
+---
+
+This is a stub commentary for the `first_session.adgroup` metric: please feel free to edit (read the
```

Comment:
These `first_session` metrics don't seem to be particularly performance-y.

Reply:
Thanks, that's a good catch! This one was mislabelled.